corehq/apps/data_cleaning/tables.py

```diff
@@ -12,20 +12,18 @@ class CleanCaseTable(BaseHtmxTable, ElasticTable):
     class Meta(BaseHtmxTable.Meta):
         pass
 
-    name = columns.Column(
-        verbose_name=gettext_lazy("Case Name"),
-    )
-    case_type = columns.Column(
-        accessor="@case_type",
-        verbose_name=gettext_lazy("Case Type"),
-    )
-    status = columns.Column(
-        accessor="@status",
-        verbose_name=gettext_lazy("Status"),
-    )
-    opened_on = columns.Column(
-        verbose_name=gettext_lazy("Opened On"),
-    )
+    @classmethod
+    def get_columns_from_session(cls, session):
+        visible_columns = []
+        for col in session.columns.all():
```

Comment:
I found it slightly confusing to make sense of this that the name `columns` is only defined on the `BulkEditColumn` relation to `BulkEditSession`, not on the session class itself. But that's not the fault of this PR.